src/fty_sensor_env.cc

```diff
@@ -41,7 +41,7 @@ bool agent_th_verbose = false;
     do { if (agent_th_verbose) zsys_debug (__VA_ARGS__); } while (0);
 
 // temporary
-#define HOSTNAME_FILE "/var/lib/fty/fty-sensor-env/agent_th"
+#define HOSTNAME_FILE "/var/lib/fty/fty-sensor-env/state"
```

Comment:
Does this need a matching action to upgrade from earlier images? From what I see, this is a copy of the hostname to self-identify, so might be just recreated after boot into new OS...